Source/Components/Touchpoints/Types/FlexCoverCarousel/Card/LogoView/MLBusinessLogoImageView.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  MlBusinessLogoImageView.swift
+//  MLBusinessComponents
+//
+//  Created by Lautaro Bonasora on 24/06/2022.
+//
+
+import Foundation
+import MLUI
+
+class MLBusinessLogoImageView: MlBusinessLogoAbstractView {
+    var imageProvider: MLBusinessImageProvider
```

Comment:
Creo que esto puede quedar así si lo inicializamos en el init con lo que nos llega por parámetro 👀 
```suggestion
    private let imageProvider: MLBusinessImageProvider
```